plugins/org.jkiss.dbeaver.ext.mysql/plugin.xml

```diff
@@ -266,6 +265,9 @@
                     <property name="@dbeaver-default-resultset.maxrows.sql" value="true"/>
                     <!--<property name="@dbeaver-default-resultset.format.datetime.native" value="true"/>-->
                     <parameter name="krb5.enabled" value="true"/>
+                    
+                    <overrideClass originalName="org.mariadb.jdbc.internal.com.send.authentication.SendPamAuthPacket"
+                                   implementationName="org.jkiss.dbeaver.ext.mysql.auth.DialogAuthPlugin2" />
```

Comment:
I couldn't find the definition for the `overrideClass` attribute along with the implementation class.

Is this PR incomplete?